rtengine/filmnegativeproc.cc

```diff
@@ -178,14 +178,14 @@ bool rtengine::RawImageSource::getFilmNegativeMedians (Coord2D topLeft, Coord2D
     x2 = CLAMP(x2 - border, 0, W);
     y2 = CLAMP(y2 - border, 0, H);
 
-    if(x1 > x2) std::swap(x1, x2);
-    if(y1 > y2) std::swap(y1, y2);
+    if (x1 > x2) { std::swap(x1, x2); }
```

Comment:
Normally we have block content on the next line like so:
```c++
if (x1 > x2) {
    std::swap(x1, x2);
}
```

Reply:
Oh, ok, i saw it used elsewhere, so i thought it was allowed. Now it's fixed

Reply:
Oh, I'll have a look...

Reply:
Our linter `astyle` took care of such constructs. The only places you can still see them are either commented out or excluded from linting. So everything is fine (kind of).

Reply:
...kind of ? If there's anyting else, please tell, i'll be happy to improve it :-)

Reply:
No, I didn't mean your code. There are many other places that could need an overhaul.